developer-docs-site/docs/index.md

```diff
@@ -112,31 +112,38 @@ Welcome! At Aptos Labs we are building a Layer 1 for everyone. This documentatio
 </div>
 </div>
 
-## Aptos devnet
+## Connect to Aptos networks
+
+Aptos offers a local testnet and online devnet and testnet, with mainnet on the way. See the [System Integrators Guide](guides/system-integrators-guide.md#networks) for a summary of the available networks and the means to connect to them.
+
 
 :::tip Aptos Devnet Resets
-The Aptos devnet is reset every Thursday Morning (PST). See the latest updates in [Aptos Discord][discord].
+The Aptos devnet is reset every Thursday. See the latest updates in Aptos [Discord][discord].
 :::info
 
-## Got a question?
+## Ask questions and offer answers
 
-Join our [Discord][discord] to speak with our developers and hop into the Aptos community. It's the best way to keep up to date with news and developments in the Aptos universe. Be sure to check pinned messages in the channels - this is where we like to keep topic-specific links, events, and more.
+Join [Discord][discord] to speak with developers and hop into the Aptos community. It's the best way to keep up to date with news and developments in the Aptos universe. Be sure to check pinned messages in the channels - this is where we like to keep topic-specific links, events, and more.
 
 Or even if you just want to stop by to say "Good Morning!". There's a little something for everyone!
 
 :::caution Be safe out there
 
-It's dangerous to go alone. Please remember, our community managers will never message or DM you first, and they will never ask you to send them money or share any sensitive, private, or personal information. If this happens to you, please report it to us in our [Discord][discord], or by sending an email to [security@aptoslabs.com](mailto:security@aptoslabs.com).
+It's dangerous to go alone. Please remember, community managers will never message or DM you first, and they will never ask you to send them money or share any sensitive, private, or personal information. If this happens to you, please report it to us in [Discord][discord], or by sending an email to [security@aptoslabs.com](mailto:security@aptoslabs.com).
```

Comment:
Lol I want a GIF of Legend of Zelda here...

Reply:
Heh! I wouldn't even mind losing the "dangerous" verbiage.